### PR TITLE
docs(developer): npm package readme files 📚

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc/README.md [hunk 1]
```diff
@@ -1,105 +1,191 @@
...
- - `kmlmc` — takes **lexical model sources** and compiles them in to a **.js**
-   file.
- - `kmlmp` — uses a `.model.kmp` file to generate a redistributable **lexical
-   model package**.
+## Install kmc
 
-`kmlmc` is intended to be used standalone, or as part of a build system. `kmlmp`
-is used only by command line tools.
+`kmc` is available as:
+* a part of Keyman Developer (Windows)
```
Comment (darcywong00) [inline developer/src/kmc/README.md:10]: Just to disambiguate from Keyman for Windows

```suggestion
* a part of Keyman Developer (Windows only)
```

Plus is it matches your heading later (integration?)

File: developer/src/kmc/README.md [hunk 2]
```diff
@@ -1,105 +1,191 @@
...
 
+### npm (Windows, macOS, and Linux)
 
-Install
--------
+kmc is also available as an npm package,
+[@keymanapp/kmc](https://npmjs.com/package/@keymanapp/kmc).
 
-Install `kmc` globally:
+You'll need [node.js](https://nodejs.org/), version 18.0 or later.
```
Comment (darcywong00) [inline developer/src/kmc/README.md:28]: Is there anything to install for WASM support?  mentioned in some readme's above for the uset parser.

Comment (mcdurdin) [reply developer/src/kmc/README.md:28]: No additional components are required -- just a reasonably modern version of Node, which this is.

File: developer/src/kmc/README.md [hunk 3]
```diff
@@ -1,105 +1,191 @@
...
 ```
 
-How to build from source
-------------------------
+**macOS**: open Keyman Configuration and drop the package khmer_angkor.kmp file
+onto the Keyman Configuration window.
 
-Run `build.sh`:
+**Android**: send khmer_angkor.kmp to your Android device, and install it from the
+hamburger menu in the Keyman app.
```
Comment (darcywong00) [inline developer/src/kmc/README.md:104]: Can reference installing-custom-packages
https://help.keyman.com/products/android/current-version/basic/installing-custom-packages

**Review by darcywong00 — APPROVED** [review]
lgtm

File: developer/src/kmc/README.md [hunk 4]
```diff
@@ -1,105 +1,191 @@
...
+km-package-install -f build/khmer_angkor.kmp
 ```
 
-How to build from source
-------------------------
+**macOS**: open Keyman Configuration and drop the package khmer_angkor.kmp file
+onto the Keyman Configuration window.
 
-Run `build.sh`:
+**Android**: send khmer_angkor.kmp to your Android device, and install it from the
```
Comment (mcdurdin) [inline developer/src/kmc/README.md:103]:
```suggestion
**Android**: send khmer_angkor.kmp to your Android device, and [install it](https://help.keyman.com/products/android/current-version/basic/installing-custom-packages) from the
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.277-beta](https://keyman.com/downloads/releases/beta/17.0.277)

**Pull request merged**